core/cmd/local_client.go

```diff
@@ -436,12 +437,67 @@ func (cli *Client) VersionDatabase(c *clipkg.Context) error {
 		return fmt.Errorf("failed to initialize orm: %v", err)
 	}
 
-	version, err := migrations.Current(orm.DB)
+	db := postgres.UnwrapGormDB(orm.DB).DB
+
+	version, err := migrate.Current(db)
 	if err != nil {
 		return fmt.Errorf("migrateDB failed: %v", err)
 	}
 
-	logger.Infof("Database version: %v", version.ID)
+	logger.Infof("Database version: %v", version)
+	return nil
+}
+
+// StatusDatabase displays the database migration status
+func (cli *Client) StatusDatabase(c *clipkg.Context) error {
+	logger.SetLogger(cli.Config.CreateProductionLogger())
+	cfg := cli.Config
+	parsed := cfg.DatabaseURL()
+	if parsed.String() == "" {
+		return cli.errorOut(errors.New("You must set DATABASE_URL env variable. HINT: If you are running this to set up your local test database, try DATABASE_URL=postgresql://postgres@localhost:5432/chainlink_test?sslmode=disable"))
+	}
+
+	orm, err := orm.NewORM(parsed.String(), cfg.DatabaseTimeout(), gracefulpanic.NewSignal(), cfg.GetDatabaseDialectConfiguredOrDefault(), cfg.GetAdvisoryLockIDConfiguredOrDefault(), cfg.GlobalLockRetryInterval().Duration(), cfg.ORMMaxOpenConns(), cfg.ORMMaxIdleConns())
```

Comment:
This `NewORM` thing is nasty, I have this card for fixing: https://app.clubhouse.io/chainlinklabs/story/14027/pull-initialization-of-db-out-of-orm-and-inject-into-newapplication

Reply:
Yeah it's a whole lot of copypasta..